crates/solidity/inputs/schema/grammar/05-expressions/02-primary-expressions/productions.yml

```diff
@@ -1,19 +1,5 @@
 # yaml-language-server: $schema=../../../../../../../codegen/schema/generated/productions.schema.json
 
-- name: "PrimaryExpression"
```

Comment:
Not sure I understand why this is removed and duplicated in the parent base expression?

I was hoping to refactor the other expressions in the future to define them in a single place, so that:

1. The definition is not duplicated, and harder to diverge/introduce bugs in the future.
2. It is easier to figure out what changed between versions, if the definition is compact.